game.exs

```diff
@@ -1,7 +1,40 @@
 defmodule GuessingGame do
 
-  def hi() do
-    IO.puts("hello world")
+  def puts_high_low(n, t) do
+    if (n < t) do
+      IO.puts("Too Low")
+    else
+      IO.puts("Too High")
+    end
   end
 
+  def win() do
+    IO.puts("That's the right number, you win!")
+  end
+
+  def get_guess() do
+    input = String.trim(IO.gets("Guess a number: "))
+    if Regex.match?(~r/^[0-9]+$/, input) do
```

Comment:
No need to worry about validation, you can actually strip it out to make the program simpler